elasticache-redis/main.tf

```diff
@@ -71,13 +72,104 @@ resource "aws_security_group" "redis" {
   tags = local.tags
 }
 
+resource "aws_kms_key" "ssm_redis_endpoint" {
```

Comment:
Do we want separate keys for each module? I wonder whether this key should belong outside this module and be used for all the Parameters from all modules? I'd be inclined to canvas some team opinion on that.

Definitely I think the same key could be used for all parameters within a module at least.

Reply:
Yes, a single kms key for all modules is a cleaner option.  But this may be a quesion for Cyber.  If one key gets compremised, then all resources are compremised.

Reply:
@ejayesh What is the process of escalating this to Cyber to get a definitive answer please?

Reply:
One key for the module will be fine